desktop/core/src/desktop/js/apps/storageBrowser/SummaryModal/SummaryModal.test.tsx

```diff
@@ -0,0 +1,97 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+
+import SummaryModal from './SummaryModal';
+import * as StorageBrowserApi from '../../../reactComponents/FileChooser/api';
+import { ContentSummary } from '../../../reactComponents/FileChooser/types';
+import { CancellablePromise } from '../../../api/cancellablePromise';
+
+Object.defineProperty(window, 'matchMedia', {
```

Comment:
See if adding to jest.init.js works, add comment that it comes from https://jestjs.io/docs/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom